theia-rust-docker/next.package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "private": true,
+    "dependencies": {
+        "@theia/callhierarchy": "next",
```

Comment:
consider: This extension could be removed - this feature does not seem to be supported by the rust Language Server and if kept there will be an entry for it in the `view` menu.

Reply:
Removed